code/modules/research/designs/borg_upgrades.dm

```diff
@@ -92,7 +92,7 @@
 	name = "Medical cyborg MK-2 upgrade"
 	desc = "Used to give a medical cyborg advanced care tools and upgrade their chemistry gripper to be able to handle pills and pill bottles."
 	id = "medical_module_surgery"
-	req_tech = list(Tc_BIOTECH = 3, Tc_ENGINEERING = 3, Tc_ANOMALY = 2)
+	req_tech = list(Tc_BIOTECH = 3, Tc_ENGINEERING = 3, Tc_ANOMALY = 3)
```

Comment:
Uh, hold up. Now you're increasing it to three from two.

Reply:
Woops, my bad. Although, isn't 3 really easy to get? I'll put it down to 2 though, was an accident.